modules/users/client/views/admin/view-user.client.view.html

```diff
@@ -2,13 +2,13 @@
   <div class="page-header">
     <div class="row">
       <div class="col-md-6">
-        <h1 ng-bind="user.username"></h1>
+        <h1 ng-bind="vm.user.username"></h1>
       </div>
       <div class="col-md-4">
         <a class="btn btn-primary" ui-sref="admin.user-edit({userId: user._id})">
           <i class="glyphicon glyphicon-edit"></i>
         </a>
-        <a class="btn btn-primary" ng-click="remove();" ng-if="user._id !== authentication.user._id">
+        <a class="btn btn-primary" ng-click="vm.remove" ng-if="vm.user._id !== vm.authentication.user._id">
```

Comment:
ng-click --> `vm.remove()`

I'm noticing this is missing from other views that have bindable methods/functions. I'll try to point them all out, but if you could go through & double check as well.